pipelines/optimus/Optimus.changelog.md

```diff
@@ -1,4 +1,24 @@
+# (next version)
+2020-04-24 (Date of Last Commit)
+
+* Added a new counting_mode parameter to Optimus workflow which enables processing of single-nuclei datasets
+
```

Comment:
Is the standard to add a blank line between each bulleted item in the changelog?  I don't think we're doing that in dsde-pipelines.

Reply:
We never talked about when making the style guide- I will have to update the guide. I changed the changelog here to be single space so that it matches dsde-pipelines.